app/controllers/application_controller.rb

```diff
@@ -227,12 +227,12 @@ def form_authenticity_token(*args)
   end
 
   # Check the user is logged in
-  def authenticated?(reason_params)
+  def authenticated?(reason_params, post_redirect: nil)
```

Comment:
Ideally we would make `reason_params` optional as its not used at all if `post_redirect` is passed in.

```suggestion
  def authenticated?(reason_params = nil, post_redirect: nil)
```

Reply:
Oh as `reason_params` is a hash this will break current implementation with `ArgumentError: unknown keywords: web, email, email_subject` and any other attributes - so perhaps not.